### ISSUE TITLE
Install Error

### ISSUE THREAD

**Issue by dbudde** [stevewithington/MuraFW1#44]
I am receiving the following error when I try to just do a straight install of a fresh Mura FW/1 master branch.

unable to deduce dot-relative path for: model (D:/vhosts/cf-instance-02/muraSite/htdocs/admin/model) root C:\ColdFusion11\cf-instance-02\bin\redirect:\index.cfm\

Altering the config for the 'diLocations' to this appears to fix the issue for me.

`	variables.framework.diLocations = 
	[
		"/" & variables.framework.package & "/admin",
		"/" & variables.framework.package & "/app1",
		"/" & variables.framework.package & "/app2",
		"/" & variables.framework.package & "/app3"
	];`

Not sure if this is expected behavior or if the base install should just work.  My setup is CF11 fully updated and Apache 2.4 using multiple CF instances and vhosts.

I have attached the full error dump.

[Error.zip](https://github.com/stevewithington/MuraFW1/files/797377/Error.zip)

**Comment by stevewithington** [comment]
@dbudde do you have a mapping for `/` in your ColdFusion Administrator? If so, remove it ... if not, try adding it. I've seen strange issues with that.

That said, you're free to modify your `config.fw1.cfm` file for your own specific environment ... so, if it works, great. Otherwise, I'm unable to reproduce your issue on my end.

**Comment by stevewithington** [comment]
@dbudde also, please double check to make sure that in your `config.fw1.cfm` file, you have the following setting:
`variables.framework.base = '/' & variables.framework.package;`

**Comment by dbudde** [comment]
I did wind up finding out that setting the `diLocations` the way I set them did not work, because it would load all the controllers from all sub systems into the parent factory.  Any named the same started to clash.  Instead, setting`diLocations=[];` wound up actually working for me.

When I have some time in the future to test setting up a base install again, I will.  I believe I do have a mapping for `this.mappings["/"]`, so I will try your suggestion of removing it.  The only problem is that there are scenarios like `expandPath("/../files/above/root/")` that will not resolve correctly for me without the mapping being in place.  Instead they resolve to a directory in my CF instance folder.  So, if you have any thoughts on that, I would appreciate it.

**Comment by dgibson666** [comment]
I'm struggling with this same issue as well. This is on a fresh Mura 7 install, CF11, Apache (win) 2.2.25. It was giving me these same 500 errors on redirects after accepting the plugin license agreement and then on any appreload. I ended up excluding admin/ in my rewrite and both cases started working again. I'm a little confused why that might be. Here's what I used for option 1 in my .htaccess:

RewriteCond %{REQUEST_FILENAME} !admin/ [NC]
RewriteRule ^(.*) /index.cfm/$1 [NC,QSA,PT]
